server/handler/upgrade_handler.go

```diff
@@ -46,9 +46,9 @@ func (h ClusterUpgradeHandler) ServeHTTP(w http.ResponseWriter, req *http.Reques
 	op := req.FormValue("op")
 	switch op {
 	case "start":
-		hasDone, err = h.startUpgrade()
+		hasDone, err = h.StartUpgrade()
```

Comment:
If there is err returned，how to handle it？

Reply:
Return error directly.

Reply:
Try again, just like in bootstrap